KSPCommon.targets

```diff
@@ -59,5 +65,51 @@
     <Message Text="@(RequiredExternalTool)" Importance="high"/>
   </Target>
 
+  <ItemDefinitionGroup>
+    <KSPVersionFile>
+      <Name>$(SolutionName)</Name>
+      <KSP_Version>1.12</KSP_Version>
+      <KSP_Version_Min>1.8</KSP_Version_Min>
+      <KSP_Version_Max>1.12</KSP_Version_Max>
+    </KSPVersionFile>
+  </ItemDefinitionGroup>
```

Comment:
Can this be conditional on it not already existing?  How would someone customize this for their own mod?

I think 1.8 is not a great idea for the min version - there are a lot of code changes in 1.10 etc and many mods are not backwards compatible.

Reply:
That's an item *definition* group, so it just sets the defaults for any other KSPVersionFile items. The metadata can be overridden there